pkg/apis/v1alpha1/cluster_template.go

```diff
@@ -66,12 +66,14 @@ type TemplateSpec struct {
 
 	// HealthRule specifies rubric for determining the health of a resource
 	// stamped by this template
```

Comment:
Nit: Can you add a full stop here? Nature of k8s documentation building is that the descriptions just run all the lines together